client/packages/invoices/src/Returns/modals/InboundReturn/useDraftInboundReturnLines.ts

```diff
@@ -1,19 +1,40 @@
 import React, { useEffect } from 'react';
-import { InboundReturnLine, RecordPatch } from '@openmsupply-client/common';
+import {
+  FnUtils,
+  InboundReturnInput,
+  InboundReturnLine,
+  InboundReturnLineInput,
+  RecordPatch,
+} from '@openmsupply-client/common';
 import { useReturns } from '../../api';
 
-export const useDraftInboundReturnLines = (stockLineIds: string[]) => {
-  const [draftLines, setDraftLines] = React.useState<InboundReturnLine[]>([]);
+export type DraftInboundReturnLine = InboundReturnLine & {
```

Comment:
InboundReturnLine would have reasonId and comment (api would), maybe best to add to MockApi now ?

Reply:
will do 🚀